### PR TITLE
[BPK-1608] Update portal to use pagewrap

### PR DESCRIPTION
**Pull request by georgegillams**
[Edit by Matteo]

This is now ready, I've just removed one file that was not part of this PR but https://github.com/Skyscanner/backpack/pull/869

### CODE REVIEW

File: packages/bpk-component-datepicker/readme.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ export default class App extends Component {
 > **Theming:** In order to theme the modal, a `renderTarget` needs to be supplied as a function which returns a DOM node
   in the scope of a `BpkThemeProvider`.
 
+> **getApplicationElement:** In order to be aligned to `microsites`, our examples use an element id value of `pagewrap`.
```
Comment (matthewdavidson) [inline packages/bpk-component-datepicker/readme.md]: How about:
`Note: the `pagewrap` element is a convention we use at Skyscanner which should "just work".`

I'd refrain from using terminology like "microsites".

Comment (georgegillams) [reply packages/bpk-component-datepicker/readme.md]: Cool. On it!

File: unreleased.md [hunk 2]
```diff
@@ -6,3 +6,11 @@
   
 - bpk-component-navigation-bar:
   - `title` prop type is now node.
+
+- bpk-component-datepicker:
```
[the remark this answers is not in the document]
Comment (georgegillams) [reply unreleased.md]: Cool, I'll get rid of it :)

File: packages/bpk-component-drawer/readme.md [hunk 3]
```diff
@@ -62,34 +59,32 @@ class App extends Component {
...
-  the root element for your application is by returning it's DOM node via the function passed to the
-  `getApplicationElement` prop (see the example above).
+> the root element for your application is by returning it's DOM node via the function passed to the
+> `getApplicationElement` prop (see the example above).
 
 > **Theming:** In order to theme the drawer, a `renderTarget` needs to be supplied as a function which returns a DOM node
-  in the scope of a `BpkThemeProvider`.
-
-> **Note:** the `pagewrap` element id is a convention we use internally at Skyscanner. In most cases it should "just work".
+> in the scope of a `BpkThemeProvider`. The `pagewrap` element id is a convention we use internally at Skyscanner. In most cases it should "just work".
```
Comment (matthewdavidson) [inline packages/bpk-component-drawer/readme.md]: This should be applied to the noe about `getApplicationElement`, not theming...

File: packages/bpk-component-modal/readme.md [hunk 4]
```diff
@@ -65,40 +60,38 @@ class App extends Component {
...
-  the root element for your application is by returning it's DOM node via the function passed to the
-  `getApplicationElement` prop (see the example above).
+> the root element for your application is by returning it's DOM node via the function passed to the
+> `getApplicationElement` prop (see the example above).
 
 > **Theming:** In order to theme the modal, a `renderTarget` needs to be supplied as a function which returns a DOM node
-  in the scope of a `BpkThemeProvider`.
-
-> **Note:** the `pagewrap` element id is a convention we use internally at Skyscanner. In most cases it should "just work".
+> in the scope of a `BpkThemeProvider`. The `pagewrap` element id is a convention we use internally at Skyscanner. In most cases it should "just work".
```
Comment (matthewdavidson) [inline packages/bpk-component-modal/readme.md]: Same here...

**Pull request merged**